sources/ClangSharp/Cursors/Decls/UsingDirectiveDecl.cs

```diff
@@ -0,0 +1,13 @@
+using ClangSharp.Interop;
+
+namespace ClangSharp
+{
+    // e.g. `using namespace std;`
+    // See: https://clang.llvm.org/doxygen/classclang_1_1UsingDirectiveDecl.html
```

Comment:
We don't currently have comment examples for anything else.

Reply:
Remove them as you see fit.